.travis.yml

```diff
@@ -1,28 +1,36 @@
 language: python
+
 python:
 - '3.6'
 - '3.5'
 - '3.4'
 
 matrix:
   include:
-    - python: 3.6
-      sudo: required
-      dist: trusty
+    - python: '3.4'
       env:
-      - TOXENV=style
+      - DNF=yum OS=centos7
```

Comment:
How about below way?
Defining not used variable for looking at it on Travis result is a little confusing me.

```
      env:
      - DNF=yum CONTAINER_IMAGE=jsmigel/centos-epel
      before_install:
      - sed -i "s@fedora@${CONTAINER_IMAGE}@" Dockerfile
```